pfio/v2/fs.py

```diff
@@ -330,18 +332,25 @@ def from_url(url: str, **kwargs) -> 'FS':
         if force_type != scheme:
             raise ValueError("URL scheme mismatch with forced type")
 
+    def _zip_check_create_not_supported():
+        if kwargs.get('create', False):
+            msg = '"create" option is not supported for Zip FS.'
+            raise UnsupportedOperation(msg)
```

Comment:
I would prefer raising  `ValueError(msg)` indicating that the application using `create` flag with ZIP is wrong.